tests/test_ThreadPool.cc

```diff
@@ -381,8 +384,16 @@ TEST_CASE("ThreadPool: count_threads()", "[ThreadPool]")
 {
     for (std::size_t i = 1; i <= 2; ++i)
     {
+        std::atomic<bool> stop{ false };
+        // Make sure the pool is removed before the atomic variable go out of scope by defining it after the atomic
```

Comment:
Typo: go -> goes (Sorry, the "suggestion" thingie is not available. Only Github knows why.)